### DISCUSSION TITLE
更新至 v3.25.1 版本后，还是无法访问阿里云盘新版本的”资源库“？

### DISCUSSION THREAD

**Discussion by jesustoachild** [AlistGo/alist#4955] [Q&A]
新版本的阿里云盘，分为资源库、备份盘，alist openwrt下更新至 v3.25.1 版本后，还是只能访问老内容的备份盘。

请问是是配置中 ”根文件夹ID“ 不对吗？填‘root’ 还是访问备份盘？

**Comment by anwen-anyi** [comment]
你没看到有个云盘类型吗？自己选择一下吖 :sob:

**Reply by jesustoachild** [reply]
搞定，多谢

**Reply by SalomonHsiun** [reply]
怎么解决的

**Reply by chenshuai19960520** [reply]
云盘类型选择了资源库 但是主页还是显示备份盘内容

**Reply by ixbcqq** [reply]
我也是这样子，云盘类型选择了资源库 但是主页还是显示备份盘内容

**Comment by Yolo0123** [comment]
自能2选1 没什么用

**Comment by anwen-anyi** [comment]
> 我也是这样子，云盘类型选择了资源库 但是主页还是显示备份盘内容

你更换了之后 右下角刷新一下呐~

**Reply by ixbcqq** [reply]
刷新啦，不仅仅是刷新，我还去后台重新挂载了然后再刷新了

**Reply by ixbcqq** [reply]
可以了，不仅仅是主文件夹的右下角刷新一下，是每进入一层文件夹的时候，都要刷新一下，一直刷新到尽头，就行了。

**Reply by anwen-anyi** [reply]
如果你着急使用  你需要那个就刷新那个就可以 不用都去刷新

**Reply by ixbcqq** [reply]
表达错误。我应该说刷新每一层目录，我也就是三四层目录而已，也很快。
